### ISSUE TITLE
Feature Request: Use custom svgs for buttons

### ISSUE THREAD

**Issue by Robstaa** [lhz516/react-h5-audio-player#24]
What do you think about the idea of making it possible to use custom images as buttons?
I was for example thinking about using a 10-sec-fast-forward button (used by Netflix on iOS for example).
![image](https://user-images.githubusercontent.com/32093573/74957511-eb746700-5407-11ea-8cc0-209a53586713.png)

Any tips on how to implement that? Thank you so much again for this great product!

**Comment by lhz516** [comment]
Currently the player uses inline svg for icons, so it's unlikely to use css to overwrite. But this is definitely a reasonable feature request. I'll add it to the roadmap.

**Comment by lhz516** [comment]
I'm proposing to add a new prop called `customIcons` and defaults to `{}`. Users can pass in custom icon elements to overwrite the  default ones. For example:
```jsx
<Player customIcons={{
    rewind: <i className="xxx" />,
    forward: <img src="xxx" />,
    play: <SomeSVGIcon />
  }}
/>
```

**Comment by Robstaa** [comment]
I love it! Would be really appreciated!!

**Comment by lhz516** [comment]
Released `2.4.0` https://github.com/lhz516/react-h5-audio-player/releases/tag/v2.4.0

**Comment by Robstaa** [comment]
You are aweome!
